### DISCUSSION TITLE
Can't get reverse options to work ":::"

### DISCUSSION THREAD

**Discussion by dbgnvan2** [st3v3nmw/obsidian-spaced-repetition#900] [Q&A]
Absolute threshold ::: The smallest amount of stimulation that can be detected by an organism.

That is not showing up as reversed - only as if I typed in "::"    I reentered the option in set for :::   but no luck. 

Suggestions?

**Comment by ronzulu** [comment]
Hi there

1. Please post your actual markdown file
2. Also post your settings file, e.g. `D:\Obsidian\Obsidian\.obsidian\plugins\obsidian-spaced-repetition\data.json`
3. What plugin version are you using

Please _attach_ the files - do not copy and paste

Cheers
Ronny

**Reply by dbgnvan2** [reply]
I think I have figured it out. The number of "items" is correct—it counts the reversed items as 2.   If I use the SKIP >> option it "skips" the reverse part of the question.  If I use the Hard - Good - Easy it does not.  Same with the "S" key.   So it makes sense that  Skip means "Skip to the next FLASH CARD.     HOWEVER,  I use "skip" as a way  of not Date Stamping so I can, on the same day do multiple reviews OF NON DATED items.     

e.g.    for a deck of 10 - Mark. 5 as Hard for review.   but Skip 5.   Some minutes later run the deck again and just to the 5.  This is my alternative the Anki "minutes" option.  

REQUEST - setting to have "skip" go to the next "item" vs. the next flashcard.   ( or Alt S or some hot key combo. ) 
Or - have a same day option vs. at least 1 day in the future.    

Thanks for listening!! 

Dave is studying like :: crazy.

**Comment by dbgnvan2** [comment]
Using the "S" or ">>" skip option skips to the NEXT FLASHCARD and bypasses the reverse part of the current flashcard.  
HOWEVER _ using the SPACE BAR to go forward DOES NOT skip the reverse item.  BUT IT DOES Date Stamp as if "hard" e.g. next day.
